src/components/repositories/Repositories.tsx

```diff
@@ -122,7 +142,7 @@ export const Repositories: React.FC = () => {
                   </Group>
                 )}
 
-                {isConnecting ? (
+                {repo.status === SERVER_STATUS.CONNECTING ? (
                   <Group mb="sm">
                     <Loader c="white" />
```

Comment:
```suggestion
                    <Loader color="white" />
```

The spinner was blue because c="white" was not the right prop to use here